faitout/faitoutlib/__init__.py

```diff
@@ -404,3 +404,21 @@ def get_stats(session):
     output['unique_ip'] = model.Connection.cnt_unique_ip(session)
 
     return output
+
+
+def get_ip_stats(session, ipaddr):
+    """ Retrieve some statistics about the current usage of faitout by a
+    given IP address.
+
+    :arg session: the session with which to connect to the faitout database.
+    :arg ipaddr: the IP address of interest.
+
+    """
+    output = {}
+
+    output['total_connections'] = model.Connection.search(
+        session, ip=ipaddr, cnt=True)
+    output['active_connections'] = model.Connection.by_ip(
```

Comment:
Why not use ``search`` for both queries or ``by_ip`` for both queries? They seem to do the same thing in this case.